addon/scripts/tokens.js

```diff
@@ -23,16 +23,27 @@
 //  r blinding factor, sjcl bignum
 function CreateBlindToken() {
     let t = newRandomPoint();
```

Comment:
I think I would either return value that's always correctly initialize or ``null``. makes error handling easier

Reply:
yes, good idea.